azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/error/ErrorCode.kt

```diff
@@ -18,6 +18,7 @@ internal class ErrorCode : ExpandableStringEnum<ErrorCode?>() {
         val CAMERA_INIT_FAILED = fromString("cameraInitiationFailure")
         val MIC_PERMISSION_DENIED = fromString("micPermissionDenied")
         val INTERNET_NOT_AVAILABLE = fromString("internetNotAvailable")
+        val MICROPHONE_BEING_USED_BY_OTHERS = fromString("microphoneBeingUsedByOthers")
```

Comment:
Please use either

MICROPHONE_NOT_AVAILABLE
or
MICROPHONE_ACCESS_FAILED

To keep more consistent with the others.